tangelo/linq/tests/test_simulator.py

```diff
@@ -139,6 +139,25 @@ def test_simulate_mixed_state_save_measures(self):
             assert_freq_dict_almost_equal(sim.all_frequencies, reference_all, 8e-2)
             assert_freq_dict_almost_equal(sim.mid_circuit_meas_freqs, reference_mid, 8e-2)
 
+    def test_simulate_mixed_state_desired_state(self):
+        """ Test mid-circuit measurement (mixed-state simulation) for all installed backends.
+        Mixed-states do not have a statevector representation, as they are statistical mixtures of several quantum states.
+        """
+
+        results = dict()
+        exact = {'11': 0.23046888414227926, '10': 0.7695311158577207}
+        for b in installed_simulator:
+            sim = get_backend(target=b, n_shots=10 ** 3)
+            results[b], _ = sim.simulate(circuit_mixed, desired_meas_result="0")
+            assert_freq_dict_almost_equal(results[b], exact, 8.e-2)
+
+    def test_desired_meas_len(self):
+        """ Test if the desired_meas_result parameter is a string and of the right length."""
+        for b in installed_simulator:
```

Comment:
This test does not depend on the choice of backend. This loop is unnecessary.

Reply:
use default backend from other tests (it's cirq)